pkg/controllers/deprovisioning/drift.go

```diff
@@ -55,7 +55,8 @@ func (d *Drift) ShouldDeprovision(ctx context.Context, c *Candidate) bool {
 	if !settings.FromContext(ctx).DriftEnabled {
```

Comment:
Thoughts on enabling this on the detection side, rather than the action side?

Reply:
Yep, we should probably rip it out here. I had thought it was only on the detection side already 👍